script/Arcueid/Day 1/01_00_ARC01_1_2.txt

```diff
@@ -42,7 +42,7 @@ As she says that, I let out an 'Ah' as everything slides into place.
 [sha:2f94cd978caa190bc82c289c8bd47007f46eb9d6]{
 -- Page 47, Offset 1274.
 -- この先輩はいつだって、誰かのために<奔走|ほんそう>している人なのだ。
-Senpai's the type that's always working hard for other people's sake.
+Senpai's the type to always work hard for other people's sake.
```

Comment:
I prefer the original because 'that's always working hard' has the implication of a constant action. 'to always works hard' sounds a bit passive, as if she waits for people to ask her for help, and then only 'works hard'.